chaos-experiments/scripts/docker-compose/verify-steady-state.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+cd $HOME/Develop/zeebe/zeebe-performance-tests/src/
```

Comment:
Could we have something, which really can be executed without depending on external resources? So like using Zbctl. We could add a simple deployment to the repository, which we just deploy and start via zbctl. I think I can add some process, which we could use also in the kubernetes tests then. This would also make sense for the existing tests, such that we are not depending on the benchmark setup.

Furthermore it makes sense to start multiple instances so we make sure we can start instances on all partitions.

Reply:
ah yes sure, I was using the zeebe-performance-tests project by Zeebe but shouldn't have added a local dependency, we can probably just use the same test workflow that is used in the k8s setup.